sdk/keyvault/keyvault-certificates/test/CRUD.test.ts

```diff
@@ -214,7 +214,7 @@ describe("Certificates client - create, read, update and delete", () => {
       base64CER,
       PEMPublicCertificate.toString()
         .split(/-----(BEGIN|END) CERTIFICATE-----/g)[2]
-        .split("\n")
+        .split(/\r?\n/)
```

Comment:
This might be a good place to use https://nodejs.org/docs/latest-v12.x/api/os.html#os_os_eol

Reply:
Oh! I didn't know about that. Do you prefer that than this regexp though?

Reply:
If the thing you're trying to split on is the end of line, then yes :smile:

I'll approve since this fixes ci in master, but if you make the EOL change the approval still stands!